packages/worker/src/workflow/reusable-vm.ts

```diff
@@ -88,7 +90,8 @@ export class ReusableVMWorkflowCreator implements WorkflowCreator {
         },
       }
     );
-    const globals = { AsyncLocalStorage, assert, __webpack_module_cache__ };
+    const SafeBuffer = { ...Buffer, allocUnsafeSlow: Buffer.alloc, allocUnsafe: Buffer.alloc };
+    const globals = { AsyncLocalStorage, assert, __webpack_module_cache__, Buffer: SafeBuffer, url };
```

Comment:
Here, `url` is the whole package's export. That means that the URL class won't be accessible directly as a global (ie. that would be `global.url.URL`). You will need to add URL and URLSearchParams directly into globals.

Reply:
I'm a bit bothered by `global` namespace pollution. I think we should wrap reexported classes and modules into a `global.__temporal_injected_modules` object, except for those that would usually be accessible in `global`.

Reply:
Removed Buffer and URL changes from this PR